src/library-authoring/routes.ts

```diff
@@ -122,18 +127,23 @@ export const useLibraryRoutes = (): LibraryRoutesData => {
       // We're inside the Units tab, so stay there,
       // optionally selecting a unit.
       route = ROUTES.UNITS;
```

Comment:
I cannot Open a Unit page from within the Units tab -- works fine from All Content tab.
(Ok to leave this bug for a follow-up PR.)

Reply:
Fixed it here: a40b18c1d47a